File: Practice2_Node/src/village_cheung_py/village_cheung_py/cheung.py
import rclpy
from rclpy.node import Node


def main(args=None):
    rclpy.init(args=args)
    node = Node("cheung")
    node.get_logger().info("大家好,我是cheung")
    rclpy.spin(node)
    rclpy.shutdown()

File: Practice2_Node/src/village_cheung/src/cheung.cpp
#include "rclcpp/rclcpp.hpp"

int main(int argc, char* argv[]){
    rclcpp::init(argc,argv);
    rclcpp::Node::SharedPtr node = std::make_shared<rclcpp::Node>("cheung");
    RCLCPP_INFO(node->get_logger(),"大家好,我是%s","cheung");
    rclcpp::spin(node);
    rclcpp::shutdown();
    return 0;

}